docs/_articles/en/v3.x/release-information.md

```diff
@@ -5,7 +5,27 @@ lang: en
 
 ### To update the plugin, please follow the linked [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
+## [v3.5.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.5.0) (09-14-2022)
 
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.4.0...v3.5.0)
+
+### Release Summary
+
+* NEW: Updates to local RetireJS Vulnerability Repository.
+* NEW \[SFGE\]: Can provide more memory while running `scanner:run:dfa` by using `--sfgejvmargs` parameter.
+See [Understanding `OutOfMemory` Java Heap Space Error](./en/v3.x/salesforce-graph-engine/working-with-sfge/#outofmemory-java-heap-space-error) for more detail.
+* FIX \[SFGE\]: If Salesforce Graph Engine errors out, partial results are now captured.
+* FIX: We fixed some under-the-hood bugs.
+
+**Merged pull requests:**
+
+- FIX: @W-11645973@: Remove noisy CliAppender warning logs when SFGE_LOG_WARNINGS_ON_VERBOSE is false [\#805](https://github.com/forcedotcom/sfdx-scanner/pull/805)
```

Comment:
@jfeingold35 I will put my suggestions into the Quip doc for John and Ramu to review. These items weren't included in our review meeting yesterday. But here is my first pass at an update:

* *IX:* @W-11645973@: We removed noisy CliAppender warning logs when SFGE_LOG_WARNINGS_ON_VERBOSE is false. [\#805](https://github.com/forcedotcom/sfdx-scanner/pull/805) (https://github.com/forcedotcom/sfdx-scanner/pull/805)
* *NEW:* @W-11646074@ We added an —sfgejvmargs parameter to the scanner:run:dfa command. [\#806](https://github.com/forcedotcom/sfdx-scanner/pull/806) (https://github.com/forcedotcom/sfdx-scanner/pull/806)
* *CHANGE:* @W-11581708@: Use the —verbose flag with scanner:run:dfa to produce productive command logs. [\#807](https://github.com/forcedotcom/sfdx-scanner/pull/807) (https://github.com/forcedotcom/sfdx-scanner/pull/807)
* *FIX:* @W-11651888@ If Salesforce Graph Engine errors out, partial results are now captured. [\#808](https://github.com/forcedotcom/sfdx-scanner/pull/808) (https://github.com/forcedotcom/sfdx-scanner/pull/808)
* *CHANGE:* @W-11261182@: Code Analyzer emits telemetry events based on messages passed from Salesforce Graph Engine. [\#811](https://github.com/forcedotcom/sfdx-scanner/pull/811) (https://github.com/forcedotcom/sfdx-scanner/pull/811)
* *NEW:* @W-11732835@: We completed routine release activities for Code Analyzer v3.5.0 release. [\#819](https://github.com/forcedotcom/sfdx-scanner/pull/819) (https://github.com/forcedotcom/sfdx-scanner/pull/819)